.evergreen/run-tests.sh

```diff
@@ -28,12 +26,23 @@ if [ "$SSL" = "yes" ]; then
    fi
 fi
 
+if [ "${SKIP_CRYPT_SHARED}" = "yes" ]; then
+   CRYPT_SHARED_LIB_PATH=""
+   echo "crypt_shared library is skipped"
+elif [ -z "${CRYPT_SHARED_LIB_PATH}" ]; then
+   echo "crypt_shared library path is empty"
+else
+   echo "crypt_shared library will be loaded from path: $CRYPT_SHARED_LIB_PATH"
+fi
+
 echo "Running tests with URI: $MONGODB_URI"
 
 # Run the tests, and store the results in a junit result file
-case "$OS" in
```

Comment:
I don't recall when this `case` construct was actually used for OS-specific behavior, but it currently serves no purpose.